v2/pkg/engine/datasource/graphql_datasource/graphql_subscription_client.go

```diff
@@ -28,7 +28,11 @@ import (
 	"go.uber.org/atomic"
 )
 
-const ackWaitTimeout = 30 * time.Second
+const (
```

Comment:
We can make this configurable in the future but passing them down from Cosmo wasn't that straightforward. I'm gonna do this in a separate PR.